apps/module-scan/src/interpreter.test.ts

```diff
@@ -60,6 +60,8 @@ test('extracts votes encoded in a QR code', async () => {
           markThresholds: { definite: 0.2, marginal: 0.17 },
         },
         testMode: true,
+        adjudicationReasons:
```

Comment:
Is this supposed to be dependent on another PR? CircleCi is complaining about centralScanAdjudicationReasons not existing in the election type?

Reply:
I extracted this commit from my work on #726, so yeah that wasn't supposed to be included.